prueba/src/Primos/Main.java

```diff
@@ -0,0 +1,15 @@
+package Primos;
+
+import java.util.Scanner;
+
+public class Main {
+
+	public static void main(String[] args) {
+		
+		Identificador identifica=new Identificador();
```

Comment:
Remember the theory about Object Oriented Programming.
The Class `Identificador` tries to check if a number is prime or not, and it has a property that is a number.
Maybe using another name for that class could be better like: `PrimeNumberChecker`, and have a function that receives a number and tells you if the number is prime or not, rather than assigning a property to that object and then using the method that will use the property assigned to it, which is more confusing.
Let me show you an example.

```java
package Primos;

import java.util.Scanner;

public class Main {
  public static void main(String[] args) {
    PrimeNumberChecker checker = new PrimeNumberChecker();
    Scanner sc = new Scanner(System.in);
    int number = scan.nextInt();
    if (checker.isPrimeNumber(number)) {
      System.out.println("The number is prime!");
    }
    else { 
      System.out.println("The number is not prime!");
    }
  }
}
```

This approach also lets the responsability of printing into the system if a number is prime or not to the Main class, and now the `PrimeNumberChecker` only checks if a number is prime or not.

This is one of the possible solutions, and there's no best solution for this, but here is another one without using another class:

```java
public class Main {
  public static boolean isPrime(number) {
    // here goes the algorithm and return if the number is boolean.
  }

  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    int number = sc.nextInt();
    if (isPrime(number)) {
      System.out.println("Number is prime");
    } else {
      System.out.println("Number is not prime");
    }
  }
}
```